scripts/irods/test/test_igroupadmin.py

```diff
@@ -104,6 +104,10 @@ def test_group_administrator_scenarios__issue_6888(self):
 
             user_with_zone = lambda user_name,zn='': '{}#{}'.format(user_name, zn if zn else self.groupadmin.zone_name)
 
+            # Test that another groupadmin (not in the group) cannot remove a member.
+
+            self.groupadmin2.assert_icommand(['igroupadmin', 'rfg', self.group, self.groupadmin.username], 'STDERR', 'CAT_INSUFFICIENT_PRIVILEGE_LEVEL');
```

Comment:
Remove semicolon at the end of the line?

Reply:
yep.